nixos/modules/services/networking/pdnsd.nix

```diff
@@ -0,0 +1,93 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.services.pdnsd;
+  pdnsd = pkgs.lib.overrideDerivation pkgs.pdnsd
+           (attrs:
+             {
+               configureFlags =
+                 [ "--enable-ipv6"
```

Comment:
We should enable IPv6 by default in the `pdnsd` package. Supporting non-legacy IP versions is a must.